imports/wasi_snapshot_preview1/fs_test.go

```diff
@@ -1147,7 +1165,7 @@ func Test_fdReaddir(t *testing.T) {
 				}
 			},
 			buf: 0, bufLen: 26, // length is long enough for one more.
-			cookie:          2,  // previous d_next
+			cookie:          1,  // previous d_next
```

Comment:
I have changed this to 1 because if you send in cookie 1, you expect to get entry 2.